src/quo2/components/list_items/received_contact_request.cljs

```diff
@@ -0,0 +1,79 @@
+(ns quo2.components.list-items.received-contact-request
+  (:require [quo.react-native :as rn]
+            [quo2.foundations.colors :as colors]
+            [status-im.utils.handlers :refer [<sub >evt]]
+            [quo2.components.avatars.user-avatar :as user-avatar]
+            [quo2.foundations.typography :as typography]
+            [clojure.string :as str]
+            [status-im.utils.utils :as utils.utils]
+            [status-im.utils.datetime :as time]
+            [status-im.i18n.i18n :as i18n]))
+
+(defn get-display-name [chat-id no-ens-name no-nickname]
+  (let [name (first (<sub [:contacts/contact-two-names-by-identity chat-id]))]
+    (if (and no-ens-name no-nickname)
+      (str (first (str/split name " ")) " " (second (str/split name " ")))
+      name)))
+
+(defn list-item [{:keys [chat-id image contact message timestamp]}]
+  (let [no-ens-name  (str/blank? (:ens-name (:content message)))
+        no-nickname  (if (= (:nickname (:names contact)) nil) true false)
+        display-name (get-display-name chat-id no-ens-name no-nickname)]
+    [rn/view {:style {:flex-direction :row
+                      :padding-top    8
+                      :margin-top     4
+                      :padding-bottom 12}}
+     [user-avatar/user-avatar {:full-name         display-name
+                               :status-indicator? true
+                               :online?           true
+                               :size              :small
+                               :profile-picture   image
+                               :ring?             false}]
+     [rn/view {:style {:margin-horizontal 8}}
+      [rn/view {:style {:flex-direction :row}}
+       [rn/text {:style (merge typography/font-semi-bold typography/paragraph-1
+                               {:color        (colors/theme-colors "#000000" "#ffffff")
```

Comment:
I think we generally avoid using hex colors in the code, and only refer to them by name or, better yet, semantically whenever possible. In this case, `(colors/theme-colors :black :white)` seems more appropriate. Same goes for a couple lines below this one, where the black and white colors are hardcoded with hex codes.